quickstarts/botolytics/config.yml

```diff
@@ -0,0 +1,35 @@
+
+slug: botolytics
+title: Botolytics
+description: |
+  ## Botolytics
+  On every well used website there are not only real users, but also many many bots. Those can be legitimate bots, like search bots, synthetic checks or other automated calls to your service. But even legitimate bots can distort your service's vitals in a way that they no longer represent your real customer's experience.
+  Being able to distinguish real traffic from bot traffic is crucial in various cases, for example when examining a marketing campaign or when analyzing your services vitals or service levels.
+  ### How to use Botolytics
+  This Quickstart works on top of your APM data. We have included the most common bots, but you can and should customize the dashboard with bots specific to your service.
+summary: |
+ Popular websites have real users and various bots. Legitimate bots, like search or checks, can distort data. Distinguishing real traffic from bots matters. It's crucial for marketing analysis and service insights. The Botolytic Dashboard helps analyze bot traffic.
+icon: logo.png
+level: Verified
+authors:
+  - Heiko Specht (New Relic)
+  - Linda Weimannn (New Relic)
+keywords:
+  - marketing
+  - bot
+  - traffic analysis
+  - apm
+  - NR1_addData
```

Comment:
```suggestion
```

Reply:
We don't want to add quickstarts to add data unless explicitly approved by a PM and Jumpstart team.